app/templates/docs/meldingsplicht-sparql.hbs

```diff
@@ -0,0 +1,145 @@
+</p>
+<p>
+  Met het volgende voorbeeld worden er 100 eerder verrichtte meldingen opgevraagd. Met de verkregen URI's kan men query's gelijk de bovenstaande opstellen.
+</p>
+<Snippet @snippetFilename="meldingsplicht-sparql/meldingen-eerste-100.sparql" />
+
+<AuHeading @level="2" @skin="2">Beschikbare gegevens</AuHeading>
+<p>
+  Het datamodel voor de inzendingen worden voorgesteld op onderstaand diagram voor Linked Data. <br>
+  In dit diagram zijn entiteiten voorgesteld met een cirkel, data properties met pijlen wijzend naar een rechthoek waar het datatype in vermeld staat en object properties met pijlen naar een nieuwe cirkel. Als er van een entiteit op het diagram geen eigenschappen bestaan zijn deze ook niet aanwezig in de data.
+</p>
+<a href="assets/images/models/MeldingsplichtSparqlAPI.svg">
+  <figure>
+    <img itemprop="thumbnail" src="assets/images/models/MeldingsplichtSparqlAPI.svg" alt="Diagram van beschikbare gegevens" />
+    <figcaption>Diagram van Linked Data van beschikbare gegevens in het endpoint</figcaption>
+  </figure>
+</a>
+<AuHeading @level="4" @skin="4">Documentstatussen inzendingen</AuHeading>
+<p>
+  Onderstaande tabel verduidelijkt de URI's die worden gebruikt om de status van een ingezonden document aan te duiden.
+</p>
+<AuTable @size="default">
+  <:header>
+    <tr>
+      <th>URI</th>
+      <th>Label</th>
+    </tr>
+  </:header>
+  <:body>
+    <tr>
+      <td><CodeInline @code="http://lblod.data.gift/concepts/79a52da4-f491-4e2f-9374-89a13cde8ecd" /></td>
+      <td>Concept</td>
+    </tr>
+    <tr>
+      <td><CodeInline @code="http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c" /></td>
+      <td>Verzonden</td>
+    </tr>
+    <tr>
+      <td><CodeInline @code="http://lblod.data.gift/concepts/f6330856-e261-430f-b949-8e510d20d0ff" /></td>
+      <td>Inzendbaar</td>
+    </tr>
+    </:body>
+</AuTable>
+
+<AuHeading @level="2" @skin="2">Prefixes</AuHeading>
+<p>
+  Onderstaande tabel beschrijft de prefixes gebruikt in dit document en in de gegevens beschikbaar in het endpoint.
+</p>
+<AuTable @size="default">
+  <:header>
+    <tr>
+      <th>Prefix</th>
+      <th>URI</th>
+    </tr>
+  </:header>
+  <:body>
+    <tr>
+      <td><CodeInline @code="rdf:" /></td>
+      <td>http://www.w3.org/1999/02/22-rdf-syntax-ns</td>
```

Comment:
I noticed the prefix URLs in `/pages-vendors/#/docs/meldingsplicht-sparql` are not clickable, unlike those in `/pages-vendors/#/docs/berichtencentrum-sparql` (e.g., [here](https://github.com/lblod/pages-vendors/pull/26/files#diff-39970087491e476fedc3eff6219bfeec625b47e79351cf602fb460913af1ec67R68)). Should we make them clickable here as well?